src/index.js

```diff
@@ -1,6 +1,13 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
+import { Provider } from 'react-redux';
 import App from './App';
+import store from './store';
```

Comment:
I indicated in `store.js` that the "conventional" export from there is a function that creates the store (and not the store itself). So here we can:

```javascript
import configureStore from './store'

const store = configureStore();
```